stages/org.osbuild.dnf

```diff
@@ -23,7 +25,7 @@ def write_repofile(f, repoid, repo):
             write_option(key, value)
 
     if "gpgkey" in repo:
-        keyfile = f"/tmp/{repoid}.asc"
+        keyfile = pathlib.Path(f.name).parent / f"{repoid}.asc"
```

Comment:
I'm unsure about this. This now requires `f` to be linked to a file that lives in some kind of a temporary directory. Before, it used `/tmp`, which is defined as a temporary location.

I think it'd be more explicit to pass a `tmpdir` into `write_repofile` and use that. 

Same for the yum stage.

Reply:
Yeah, good point. Fixed in commit d916eac.